DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
+human involved at first for moving the data and making sure all the
+pieces work. ~ It should also be tested for when things are working more
+as a service, and without a human involved for an automatic data archive
+process. ~
 
-        \\ \midrule
-    \end{longtable}
 
+  \subsubsection{Software Version/Baseline}
+    The DBB version is 1.0, and can be found in the github repository when
+it has been fully developed. ~ The current DBB is a script for scraping
+data from the header file and getting the file name and the metadata
+required along with providence. ~ At the time of this writing, the DBB
+use cases and requirements are just being written. ~ ~The database
+component is Oracle and the ingest file systems and output file systems
+for DBB are in GPFS at the LDF.~~
 
 
-\newpage
+  \subsubsection{Configuration}
+    The configuration for this test is the data is located in a file system
+somewhere. ~It is transferred to NCSA into a the LSST GPFS file system.
+~ Files that are needed in the DBB are moved to the DBB ingest file
+system. ~ the data backbone gathers the info from the headers and file
+name. ~it inserts rows into the DBB consolidated DB environment and
+moves the file to the DBB file systems that are used by the LSP to once
+again find the raw image file to be viewed/displayed for further
+analysis. ~
 
-\section{Test Results}
-\label{sect:testresults}
 
-\subsection{Overview of the Test Results}
-\label{sect:overview}
+  \subsubsection{Test Cases in LVV-C8 Test Cycle}
 
-\subsubsection{Summary Table}
-\label{sect:summarytable}
 
+    \paragraph{Test Case LVV-T284 }\mbox{}\\
 
-\begin{longtable} {|p{0.2\textwidth}|p{0.2\textwidth}|p{0.6\textwidth}|}
-\hline
-{\bf TEST CASE ID} & {\bf PASS/FAIL} & {\bf COMMENTS} \\\hline
-LVV-T284 & Not Run & \\\hline
-\end{longtable}
+Open  {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T284}{test case} } in Jira.
 
-\subsubsection{Overall Assessment}
-\label{sect:overallassessment}
+    This test will check:
 
+\begin{itemize}
+\tightlist
+\item
+  The successful integration of the Pathfinder components with the CCOB;
+\item
+  That the file can then be ingested into the DBB and be retrieved for
+  further analysis;
+\end{itemize}
 
-\subsubsection{Recommended Improvements}
-\label{sect:recommendations}
 
-\subsection{Detailed Test Results}
-\label{sect:detailedtestresults}
+    {\bf Preconditions}:\\
+    None.
+
+
+    Execution status: {\bf Not Executed }
+
+    Final comment:\\Need to have the CCOB raw data files pinpointed to be the ones that is
+wanted to be placed in the DBB at NCSA.~~
+
+
+
+    Detailed step results:
+
+    \begin{longtable}{p{1cm}p{2cm}p{13cm}}
+    \hline
+    {Step} & \multicolumn{2}{c}{Description, Results and Status}\\ \hline
+      1 & Description &
+
+      \begin{minipage}[t]{13cm}{\footnotesize
+      CCOB device directs a human to where a raw file is wanted to be stored
```

Comment:
How does the test start? Will the CCOB spontaneously light up and direct the human to a file? Or does somebody need to take data on the CCOB? Or press a button? Or...?

Reply:
This has to be reviewed in the test spec...

Reply:
fixed wording